src/components/Player/TableFilterForm/index.jsx

```diff
@@ -28,7 +28,7 @@ const Styled = styled.div`
 .showForm {
   overflow: hidden;
   transition: max-height 0.2s;
-  max-height: 1000px;
+  max-height: 1200px;
```

Comment:
will this just fail if we add more forms?  What if we just remove it?

Reply:
OK, removing it works. I tested it.